documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,263 @@
+
+In the disruption, the delay can be found in the list of "impacted_stops" with the departure/arrival status set to "delayed". See the [disruption](#disruption) objects section for its full content and description.
+
+<div></div>
+### Journeys
+
+``` shell
+# Request example for /journeys
+http://api.navitia.io/v1/coverage/<coverage>/journeys?from=<origin>&to=<destination>&data_freshness=realtime
+```
+
+``` shell
+# Extract of the public transport section of the response /journeys
+
+    "sections": ⊖[
+       ⊖{
+            "additional_informations": ⊕[1 item],
+            "arrival_date_time": "20190529T205000",
+            "base_arrival_date_time": "20190529T204500",
+            "base_departure_date_time": "20190529T160000",
+            "co2_emission": ⊕{2 items},
+            "data_freshness": "realtime",
+            "departure_date_time": "20190529T160000",
+            "display_informations": ⊕{13 items},
+            "duration": 21180,
+            "from": ⊕{5 items},
+            "geojson": ⊕{3 items},
+            "id": "section_0_0",
+            "links": ⊕[6 items],
+            "stop_date_times": ⊖[
+               ⊕{7 items},
+               ⊖{
+                    "additional_informations": [],
+                    "arrival_date_time": "20190605T194700",
+                    "base_arrival_date_time": "20190605T193200",
+                    "base_departure_date_time": "20190605T193400"
+                    "departure_date_time": "20190605T194900",
+                    "links": [],
+                    "stop_point": ⊕{7 items},
+                },
+               ⊕{7 items},
+               ⊕{7 items},
+            ]
+            "to": ⊕{5 items},
+            "type": "public_transport",
+        }
+    ]
+```
+
+
+The status of the journey is `SIGNIFICANT_DELAYS`. In a public transport section of the response, "base_arrival_date_time"/"base_departure_date_time" represent the scheduled arrival/departure time without taking into account the delay whereas "arrival_date_time"/"departure_date_time" are the actual arrival/departure time, after the delay is applied. The delay can also be observed for every stop_point of the journey with the same parameters in "stop_date_times".
+If the parameter "data_freshness" is set to "base_schedule",  "base_arrival_date_time"/"base_departure_date_time" = "arrival_date_time"/"departure_date_time".
+
+A list of the disruptions impacting the journey is also present at the root level of the response.
+
+<aside class="notice">
+For a disruption to be present in the response, the request has to be made during its application period.
+</aside>
+
+<div></div>
+### Departures
+
+``` shell
+# Request example for /departures
+http://api.navitia.io/v1/coverage/<coverage>/physical_modes/<physical_mode>/stop_points/<stop_point>/departures?from_datetime=<from_date>&data_freshness=realtime
```

Comment:
Please switch the example call to stop_area (to be consistent with trip-add and stop_add when they will be documented)

Reply:
As discussed in refinement, indeed!